### DISCUSSION TITLE
Calculation of PTC value for CCS

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#100] [Q&A: Data]
Hi ReEDS team! I was wondering how the ptc_value ($/MWh) in inputs/incentives/incentives_annual.csv is determined based on the section 45Q credit ($/metric ton). I am trying some calculations to get the same number of ptc_value in incentives_annual.csv, but was not able to do so. The equations I am using is as follows.

ptc_value ($/MWh) = 45Q credits per year ($/metric ton) / capture rate (metric ton/MWh).

Where (1) the 45Q credits per year represent $50/metric ton of CO2 storage in saline reservoir, and (2) the capture rate (metric tons/MWh) is obtained from run/[case_name]/input_cases/rep_[case_name].gdx.

Did I miss something?

**Comment by wesleyjcole** [comment]
This ptc value was calculated using the heat rate, fuel emission rate, and 45Q credit value.  The values were calculated using the ATB 2020 natural gas specifications for heat rate.

**Reply by wanyingw193** [reply]
Get it! Thank you so much!

Could I ask why the 45Q credit is added as a PTC and not in the same way as the emit_tax in the objective function? Thanks!
![image](https://user-images.githubusercontent.com/78582696/156449724-33d3550b-0362-4458-95f0-91edaa61477a.png)
